modules/service/src/main/resources/db/migration/V1047__gmos_long_slit_acq_roi.sql

```diff
@@ -0,0 +1,60 @@
+
+CREATE TYPE e_gmos_long_slit_acquisition_roi AS enum(
+  'Ccd2Stamp',
+  'Ccd2',
+  'Stamp',
+  'FullCcd2'
+);
+
+-- Add a column to keep up with the explicitly specified GMOS North acquisition
+-- ROI.
+ALTER TABLE t_gmos_north_long_slit
```

Comment:
is this working? often I need to split creating an enum and using it in two migrations

Reply:
I tested it with dev data and it was fine.